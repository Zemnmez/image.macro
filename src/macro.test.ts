import pluginTester from 'babel-plugin-tester';
import plugin from 'babel-plugin-macros';

pluginTester({
    plugin,
    snapshot: true,
    babelOptions: { filename: __filename},
    tests: [
        `import macro from '../../image.macro';
        const img = macro('${__dirname}/../test.png');`
    ]
})