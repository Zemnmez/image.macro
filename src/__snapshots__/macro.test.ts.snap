// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`macros 1. macros: 1. macros 1`] = `

import macro from '../../image.macro';
        const img = macro('/mnt/c/Users/thoma/Documents/devel/image.macro/src/../test.png');

      ↓ ↓ ↓ ↓ ↓ ↓

const img = import(".cache/image.macro/component..9a561f2052.jsx");


`;
